Detectors/EMCAL/calibration/src/EMCALTimeCalibData.cxx

```diff
@@ -55,7 +56,7 @@ void EMCALTimeCalibData::merge(const EMCALTimeCalibData* prev)
   mTimeHisto += prev->getHisto();
 }
 //_____________________________________________
-bool EMCALTimeCalibData::hasEnoughData() const
+bool EMCALTimeCalibData::hasEnoughData(const int minNEntries) const
```

Comment:
For primitive datatypes the const specifier is usually useless, they are in any case copied by-value so whatever modification a function would do doesn't matter outside, and for within the function constant is a lie as the value is set at runtime.

Reply:
done